build.gradle

```diff
@@ -2,35 +2,36 @@
 buildscript {
 
     ext {
-        kotlin_version = "1.4.32"
+        kotlinVersion = '1.5.21'
         retrofitVersion = '2.9.0'
-        navigationVersion = "2.3.4"
+        navigationVersion = '2.3.4'
         koinVersion = '2.0.1'
-        parcelerVersion = "1.1.13"
-        roomVersion = '2.2.6'
-        firebaseAnalyticsVersion = '18.0.2'
-        firebaseCrashlyticsVersion = '17.4.1'
-        firebaseMessagingVersion = "20.2.0"
-        firebaseConfigVersion = "19.1.4"
-        adapterDelegatesVersion = "4.2.0"
-        glideVersion = "4.9.0"
-        navigationVersion = "2.3.4"
+        parcelerVersion = '1.1.13'
+        roomVersion = '2.3.0'
+        firebaseAnalyticsVersion = '19.0.0'
+        firebaseCrashlyticsVersion = '18.2.1'
+        firebaseMessagingVersion = '20.2.0'
+        firebaseConfigVersion = '19.1.4'
+        adapterDelegatesVersion = '4.2.0'
+        glideVersion = '4.9.0'
+        navigationVersion = '2.3.5'
```

Comment:
We should use the latest version(2.4.0-alpha07) because it introduces support for multiple backstack which would be useful.

Reply:
I didn't know if it was ok to use an alpha version of the package. It's updated now.